test/hotspot/jtreg/serviceability/jvmti/HiddenClass/libHiddenClassSigTest.cpp

```diff
@@ -47,6 +51,24 @@ static bool failed = false;
     return; \
   }
 
+struct MonitorLock {
```

Comment:
jvmti.common has RawMonitorLocker
(and LOG)
could they be used in test?
or even better to just use 
std::atomic<bool>
if it is enough.
The atomic are not used in hotspot but no limitation about using them in test code.

Reply:
Good point, updated to RawMonitorLocker.
I left LOG0/LOG1/LOG2 as is to avoid a lot of changes in the test
std::atomic<bool> is not enough in the case.